app.py

```diff
@@ -0,0 +1,99 @@
+"""
```

Comment:
would you be able to put this into a folder called neso-solar-consumer/
perhaps fetch_data should also be moved there

Reply:
Thank you for the feedback! I’ll move the code into a folder named `neso-solar-consumer` and include `fetch_data` there as well. I had a similar thought, so I’m glad we’re on the same page!"